src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -844,157 +577,38 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
-  } = useShowHide(DEFAULT_SHOW_USER_STATUSES);
-
-  // Because this is not using the useForm validation. The use effect needs to manually open the dropdown with user statuses.
-  useEffect(() => {
-    if (!sovereignVenue) return;
-
-    const venueUserStatuses = sovereignVenue?.userStatuses ?? [];
-    const venueShowUserStatus =
-      sovereignVenue?.showUserStatus ?? DEFAULT_SHOW_USER_STATUSES;
-
-    setUserStatuses(venueUserStatuses);
-
-    if (venueShowUserStatus) {
-      showUserStatuses();
-    } else {
-      hideUserStatuses();
-    }
-  }, [hideUserStatuses, showUserStatuses, sovereignVenue]);
-
-  const removeUserStatus = (index: number) => {
-    const statuses = [...userStatuses];
-    statuses.splice(index, 1);
-    setUserStatuses(statuses);
-  };
-
-  const addUserStatus = () =>
-    setUserStatuses([
-      ...userStatuses,
-      { status: "", color: DEFAULT_USER_STATUS.color },
-    ]);
-
-  const updateStatusColor = (color: string, index: number) => {
-    const statuses = [...userStatuses];
-    statuses[index] = { color, status: statuses[index].status };
-    setUserStatuses(statuses);
-  };
-
-  const updateStatusText = (
-    event: React.FormEvent<HTMLInputElement>,
-    index: number
-  ) => {
-    const statuses = [...userStatuses];
-
-    const userStatusExists = statuses.find(
-      (userStatus) => userStatus.status === event.currentTarget.value
-    );
-
-    // @debt Move user statuses to useForm with useDynamicInput, add schema for this validation instead
-    if (userStatusExists) {
-      setError("User statuses", {
-        type: "manual",
-        message: "User status already exists.",
-      });
-      setFormError(true);
-    } else {
-      setError("", "");
-      setFormError(false);
-    }
-
-    statuses[index] = {
-      color: statuses[index].color,
-      status: event.currentTarget.value,
-    };
-    setUserStatuses(statuses);
-  };
-
   return (
-    <form
-      className="full-height-container"
-      onSubmit={handleSubmit((vals) =>
-        onSubmit(vals, userStatuses, hasUserStatuses)
-      )}
-    >
+    <form className="full-height-container" onSubmit={handleSubmit(onSubmit)}>
       <input type="hidden" name="template" value={templateID} ref={register} />
       <div className="scrollable-content">
         <h4 className="italic" style={{ fontSize: "30px" }}>{`${
           editing ? "Edit" : "Create"
         } your ${templateType}`}</h4>
-        <p
-          className="small light"
-          style={{ marginBottom: "2rem", fontSize: "16px" }}
-        >
-          You can change anything except for the name of your venue later
-        </p>
+        {renderHelper(
+          "You can change anything except for the name of your venue later"
+        )}
 
         {renderVenueNameInput()}
         {renderTaglineInput()}
         {renderDescriptionInput()}
-        {renderRestrictToAdultsInput()}
 
         {renderBannerPhotoInput()}
+        {renderHelper("1:1 ratio recommended")}
```

Comment:
This should be below the logo, not the banner.

Reply:
fixed in def392d3865e2f8d0eaf232f49ef621de2d48e48